lusidtools/cocoon/cocoon.py

```diff
@@ -91,7 +91,9 @@ def get_alphabetically_first_identifier_key(
             first_unique_identifier_alphabetically = unique_identifiers_populated[0]
             return f"{first_unique_identifier_alphabetically}: {instrument.identifiers[first_unique_identifier_alphabetically].value}"
 
+        # If scope is not defined set to default scope
         return api_factory.build(lusid.api.InstrumentsApi).upsert_instruments(
+            scope=kwargs.get("scope") if kwargs.get("scope") != None else "default",
```

Comment:
I think you can just do 
```
scope=kwargs.get("scope"), "default")
```

as `kwargs` is just a dict

Reply:
![Screenshot 2022-06-27 at 10 03 53](https://user-images.githubusercontent.com/43269117/175902331-0fbe017a-41f3-4e22-a6b1-d1d816588eda.png)